playbooks/archive_artifacts.yml

```diff
@@ -0,0 +1,77 @@
+- hosts: "{{ hosts }}"
+  tasks:
+    - name: Create temporary directory to store artifacts
+      file:
+        path: "{{ artifacts_path }}"
```

Comment:
Without creating "{{ artifacts_path }}" rsync will fail, any required subdirectories will be created by rsync. So, this can be done here but it's not necessary.